src/components/Dashboard/Link.test.js

```diff
@@ -5,7 +5,16 @@ import configureStore from 'redux-mock-store';
 
 import { initialState } from 'reducers/dashboard';
 import ConnectedLink, { DashboardLink } from './Link';
-import { withMemoryRouter } from 'utils';
+import { withMemoryRouter } from 'utils/testing';
+
+jest.mock('components/Pagination/Pagination', () => {
```

Comment:
Why does this component require Pagination to be mocked?

Reply:
It doesn't, good catch!